recipes/libibverbs-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: libibverbs-cos6-x86_64
+  version: 1.1.8
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libibverbs-1.1.8-4.el6.x86_64.rpm
+    sha256: 706e5770d5c3e6c06f7c47a88936edbee56a003f31186ea4dd7e1de8b87903ad
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/libibverbs-1.1.8-4.el6.src.rpm
+    folder: source
```

Comment:
Currently there is no good way to determine the checksum of the source RPM. Please see issue ( https://github.com/conda/conda-build/issues/3552 ) for details.

Reply:
I've gone ahead and dropped the source RPM as it is not used.